pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -79,9 +79,7 @@ def configure_logger(self, logger: Union[bool, LightningLoggerBase, Iterable[Lig
         if isinstance(logger, bool):
             # default logger
             self.trainer.logger = (
-                TensorBoardLogger(
-                    save_dir=self.trainer.default_root_dir, version=SLURMEnvironment.job_id(), name="lightning_logs"
```

Comment:
Sure, I am fine with making the default `lightning_logs`. No strong preference either way. What do other people think? 

> since now it will create logs like this in the current directory by default:

It depends on the logger, for TB by default it will create logs in `default/version_0`, `default/version_2`:
https://github.com/PyTorchLightning/pytorch-lightning/blob/86b177ebe5427725b35fde1a8808a7b59b8a277a/pytorch_lightning/loggers/tensorboard.py#L88

Reply:
I'm not in favour of changing the default directory of the default logger as it'll affect quite a lot of users.

Reply:
Yes, let's make `lightning_logs` the default for all loggers. This would be expected behaviour by the users.

Reply:
Sg! Will update this PR